lib/resources/majors.tsx

```diff
@@ -5164,6 +5164,201 @@ const bsNeuro: Major = {
+  distributions: [
+    {
+      name: 'Statistics',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description:
+        'These courses expose students to the foundations of statistics that are used extensively in business ' +
+        'decision-making. These topics include correlation, estimation, hypothesis testing, linear regression, prediction, ' +
+        'and forecasting. Students may take either two elementary statistics courses or one intermediate/advanced calculus-based' +
+        ' statistics course from the list below.',
+      criteria:
+        'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]^OR^' +
+        'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.430[C]^OR^EN.560.240[C]',
+      fine_requirements: [
+        {
+          required_credits: 8,
+          description:
+            '<b>Option One: Two Elementary Statistics Courses.</b> Credit earned for AP Statistics (equivalent to EN.553.111)' +
+            ' will satisfy the first of the two required courses.<br />' +
+            'First course (select one): <br/>' +
+            'AS.230.205 Introduction to Social Statistics' +
+            '<br /> <i>OR</i> <br />' +
+            'AS.280.345 Public Health Biostatistics' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.111 Statistical Analysis I <br/>' +
+            'Second course (select one) <br/>' +
+            'EN.553.112 Statistical Analysis II' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.380 Decision Analytics <br/>',
+          criteria:
+            'AS.230.205[C]^OR^AS.280.345[C]^OR^AS.280.345[C]^OR^EN.553.111[C]^OR^EN.553.112[C]^OR^EN.661.380[C]',
+        },
+
+        {
+          required_credits: 3,
+          description:
+            '<b>Option Two: One Intermediate/Calculus-Based Course (must have Calculus as a prerequisite)</b><br />' +
+            'EN.553.211 Probability and Statistics for the Life Sciences' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.310 Probability & Statistics for the Physical Sciences & Engineering' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.311 Probability and Statistics for the Biological Sciences and Engineering' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.553.430 Introduction to Statistics' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.560.240 Uncertainty, Reliability and Decision-making',
+          criteria:
+            'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^EN.553.430[C]^OR^EN.560.240[C]',
+        },
+      ],
+    },
+    {
+      name: 'Entrepreneurship and Management Fundamentals',
+      required_credits: 9,
+      min_credits_per_course: 3,
+      description:
+        'Students must complete three fundamental courses in entrepreneurship and management.',
+      criteria:
+        'EN.660.105[C]^OR^EN.660.250[C]^OR^EN.660.203[C]^OR^EN.661.110[C]^OR^EN.661.250[C]',
```

Comment:
ok i know nothing abt writing majors so maybe i'm wrong but shouldn't it be like (105 or 250) and 203 and (110 or 250)?

Reply:
Josephine's right, this criteria is fine but the fine distributions below should be three categories: 

 
<img width="825" alt="Screenshot 2023-09-22 at 3 34 21 PM" src="https://github.com/uCredit-Dev/ucredit_frontend_typescript/assets/90944895/ec4ebc3a-14af-4347-83fe-2e7d12e7ff8e">